### PR TITLE
Lesson 13

### PR DESCRIPTION
**Pull request by Anukrei**

### CODE REVIEW

File: lesson-13/main.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  let itemArr;
+  for (let i = 0; i < arr.length; i++) {
+    itemArr = arr[i];
+    hotels = document.createElement("div");
+    hotels.innerHTML = `
+	 	<img class="hotel11" src="${itemArr.imageUrl}" alt="${itemArr.name}">
+		 <div class="name_hotel">${itemArr.name}</div>
+		 <div>${itemArr.city}, ${itemArr.country}</div>`;
+    homes.append(hotels);
+  }
```
Comment (Dronom) [inline lesson-13/main.js:40]: I would use forEach here, but okay

File: lesson-13/main.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      slidesToShow: 4,
+      slidesToScroll: 4,
+      infinite: true,
+      arrows: true,
+    });
+  });
+}
+
+document.addEventListener("DOMContentLoaded", async function () {
+   homes = await eventSearch();
```
Comment (Dronom) [inline lesson-13/main.js:53]: why do you assign a variable to the result of a function that returns nothing?

File: lesson-13/main.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+});
+
+async function eventSearch() {
+  const searchResults = sessionStorage.getItem(homes);
```
Comment (Dronom) [inline lesson-13/main.js:57]: homes should be in quotes. You are referring to the key

File: lesson-13/main.js (hunk 2 above)
Comment (Anukrei) [reply lesson-13/main.js]: исправил

File: lesson-13/main.js (hunk 3 above)
Comment (Anukrei) [reply lesson-13/main.js]: исправил

**Review by Dronom — APPROVED** [review]

**Pull request merged**